src/test/java/nl/fuchsia/model/FeitTest.java

```diff
@@ -0,0 +1,14 @@
+package nl.fuchsia.model;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FeitTest {
+    @Test
+    public void testToString() {
```

Comment:
Goed dat je het doet, maar de toString methode is meestal niet belangrijk voor de werking van de applicatie, dus dat hoef je niet te testen.

Reply:
In overleg met Merijn gedaan, we zijn samen mijn classes langs gegaan en vervolgens heb ik test gemaakt.